### DISCUSSION TITLE
Is there a way to use only storage in appwrite without user because i am getting 404 when firing the request

### DISCUSSION THREAD

**Discussion by scshiv29-dev** [appwrite/appwrite#4576] [Q&A] [answered]
Is there a way to use only storage in appwrite without user login . I just want to use the storage to store some files but i am unable to make request it returns 404 without a jwt. which i dont have because i dont want user login.

**Comment by stnguyen90** [comment, marked as answer]
Yes, but you'll need to give everyone (`any`) read access to the bucket. 

Refer to our [Permissions docs](https://appwrite.io/docs/permissions) for more info.
